test/functional/wallet_groups.py

```diff
@@ -103,13 +103,25 @@ def run_test(self):
         self.nodes[0].sendtoaddress(addr_aps, 1.0)
         self.nodes[0].sendtoaddress(addr_aps, 1.0)
         self.nodes[0].generate(1)
-        txid4 = self.nodes[3].sendtoaddress(self.nodes[0].getnewaddress(), 0.1)
+        self.sync_all()
+        with self.nodes[3].assert_debug_log(['Fee non-grouped = 2820, grouped = 4160, using grouped']):
+            txid4 = self.nodes[3].sendtoaddress(self.nodes[0].getnewaddress(), 0.1)
         tx4 = self.nodes[3].getrawtransaction(txid4, True)
         # tx4 should have 2 inputs and 2 outputs although one output would
         # have been enough and the transaction caused higher fees
         assert_equal(2, len(tx4["vin"]))
         assert_equal(2, len(tx4["vout"]))
 
+        addr_aps2 = self.nodes[3].getnewaddress()
+        [self.nodes[0].sendtoaddress(addr_aps2, 1.0) for _ in range(5)]
+        self.nodes[0].generate(1)
+        with self.nodes[3].assert_debug_log(['Fee non-grouped = 5520, grouped = 8240, using non-grouped']):
+            txid5 = self.nodes[3].sendtoaddress(self.nodes[0].getnewaddress(), 2.95)
```

Comment:
Test fails here:

https://cirrus-ci.com/task/6483042137014272?command=ci#L5047

```
 test  2020-08-17T11:25:30.841000Z TestFramework (ERROR): JSONRPC error 
                                   Traceback (most recent call last):
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_framework.py", line 118, in main
                                       self.run_test()
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/wallet_groups.py", line 119, in run_test
                                       txid5 = self.nodes[3].sendtoaddress(self.nodes[0].getnewaddress(), 2.95)
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/coverage.py", line 47, in __call__
                                       return_val = self.auth_service_proxy_instance.__call__(*args, **kwargs)
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/authproxy.py", line 146, in __call__
                                       raise JSONRPCException(response['error'], status)
                                   test_framework.authproxy.JSONRPCException: Insufficient funds (-6)

Reply:
![image](https://user-images.githubusercontent.com/2415484/90424067-cd897580-e0bd-11ea-9160-34cdf3183640.jpeg)

This may be a second one? I saw it at line 106.